packages/terra-base/README.md

```diff
@@ -6,7 +6,7 @@
 The `terra-base` component is a core feature for any apps built with Terra UI. The base component handles two concerns, application internationalization and global styles. Read the [terra-base docs]((https://github.com/cerner/terra-core/tree/master/packages/terra-base/docs)) to learn more about the component.
 
 - [Getting Started](#getting-started)
-- [Documentation](https://github.com/cerner/terra-core/tree/master/packages/terra-base/docs)
+- [Documentation](https://engineering.cerner.com/terra-ui/components/terra-base/base/base)
```

Comment:
Link to React-Intl Provider has been moved to : https://github.com/formatjs/react-intl/blob/master/docs/Components.md  from https://github.com/formatjs/react-intl/wiki/Components#intlprovider same needs to updated in terra-base as well.